packages/react-storage/src/components/StorageBrowser/controls/hooks/__tests__/useTitle.spec.tsx

```diff
@@ -0,0 +1,40 @@
+import { useControlsContext } from '../../../controls/context';
+import { useTitle } from '../useTitle';
+
+jest.mock('../../../controls/context');
+
+describe('useTitle', () => {
+  const data = {
+    title: 'ShinyNewTitle',
+  };
+
+  const emptyData = {};
+
+  const actionsConfig = {
+    isCancelable: true,
+    type: 'BATCH_ACTION',
+  };
```

Comment:
Isn't this marked optional now? Let's just drop it throughout this test?

Reply:
Oh yeah, just realized it's optional now. Removed it!